src/live-programming.rst

```diff
@@ -26,6 +26,64 @@ Interactive Debuggers
 Program Visualization
 ---------------------
```

Comment:
Let's move all of the Program Visualization stuff here into the Program Comprehension chapter, instead of splitting it like this. If there is a "live" program visualization system, i.e. one that keeps visualizations updated during the editing process, then you can include a section here called "Live Program Visualization" to specifically mention that. However, the examples below seem to all be standard program visualization tools that are not deeply integrated into editing.